src/nr_openai_observability/build_events.py

```diff
@@ -250,6 +229,24 @@ def build_embedding_error_event(request, error):
         "error_type": error.error.type,
         "error_code": error.error.code,
         "error_param": error.error.param,
+        **get_trace_details(),
     }
 
     return embedding
+
+
+def get_trace_details():
+    span_id = newrelic.agent.current_span_id()
+    trace_id = newrelic.agent.current_trace_id()
+    transaction_id = (
+        newrelic.agent.current_transaction().guid
+        if newrelic.agent.current_transaction() != None
+        else None
+    )
+    return {
+        "span_id": span_id,
+        "trace_id": trace_id,
+        "trace.id": trace_id,  # Legacy value from SDK
```

Comment:
`trace_id` is the attribute that the official agent will support. `trace.id` is what the SDK has been collecting. Recording both versions for now for backwards compatibility